nexus/lib/pwscf_analyzer.py

```diff
@@ -187,6 +187,25 @@ def analyze(self):
             #end if
         #end try
 
+        try:
+            fermi_energies = []
+            for l in lines:
+                if l.find('Fermi energy')!=-1:
+                    fermi_energies.append( eval( l.split('is')[1].split()[0] ) )
```

Comment:
Prefer "float" over "eval".  I try to avoid "eval" as much as possible because it can be used to execute arbitrary Python code from text.